package.json

```diff
@@ -6,7 +6,10 @@
     "tree-sitter",
     "semmle-ql"
   ],
-  "repository": "https://github.com/samlanning/tree-sitter-ql",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/samlanning/tree-sitter-ql.git"
```

Comment:
Can you update this to point to the new location (i.e. this repository)

Reply:
done